### DISCUSSION TITLE
claim rewards via python RPC

### DISCUSSION THREAD

**Discussion by hajes** [Chia-Network/chia-blockchain#17487] [Support]
hi, I have created a python script that 1st day of month claims rewards, export all transactions, do some calculations per plot/TiB basis, and send rewards to cold wallet.

I have managed all except `claiming rewards`, I can't find it in RPC documentation, or it is under different name to `chia plotnft`.

so far I have found `pw_absorb_rewards` in ` [documentation](https://docs.chia.net/wallet-rpc/#pw_absorb_rewards). I am guessing that is something else.

The kludge so far is python `subprocess.call` with the command `. ~/chia-blockchain/activate && chia plotnft claim -i 14 -m 0.00001`

any suggestions, please?

P.S. could be a solution to simply `send_transaction` from pooling to main wallet???

**Comment by BrandtH22** [comment]
Hey @hajes , that pw_absorb_rewards should be the one you are looking for 👍 
Let us know if you run into issues or have any questions

**Reply by hajes** [reply]
it worked. without fee, it got stuck again. deleted unconfirmed, and `pw_absorb_rewards` again with fee. thanks guys
